### DISCUSSION TITLE
Hacked Signal app help

### DISCUSSION THREAD

**Discussion by Zoey8581** [community/community#69259] [New to GitHub] [answered]
I am new here. I believe I just found hackers code to get into my Signal App.  CAN YOU PLEASE LOOK AT SCREENSHOTS AND TELL ME?
![Screenshot_20231004_185717_HTML Viewer](https://github.com/community/community/assets/146785212/015bc4e5-13e7-4b4b-8b2f-32314ffd1f7b)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Zoey8581, Thanks for reaching out! 

We're sorry to hear you're experiencing this issue.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!

**Reply by Wwrrdnmld** [reply]
کی فیلتر میخوام

**Comment by efake3220** [comment]
Bhaiya mujhe bhi hacking seekhna hai
